app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ActiveRecord::Base
+  has_many :comments, dependent: :destroy, foreign_key: 'author_id'
+  has_many :posts, dependent: :destroy, foreign_key: 'author_id'
+  has_many :likes, dependent: :destroy, foreign_key: 'author_id'
+
+  def most_recent_posts
+    Post.where(author_id: id).order(created_at: :desc).limit(3)
```

Comment:
- [ ] (Optional) Currently, the `id` in the `author_id: id` is undefined because it is not initialized anywhere. You can decide to pass in the `id` as a parameter of the method and then you will provide the id when you want to call it or you follow the option the previous reviewer gave.

```ruby on rails
def most_recent_posts(id)
   Post.where(author_id: id).order(created_at: :desc).limit(3)
 end
```
or 
```ruby on rails
posts.order('created_at Desc').limit(3)
```